components/raftstore/src/store/worker/metrics.rs

```diff
@@ -165,6 +165,21 @@ lazy_static! {
         "Bucketed histogram of full compaction for the storage."
     )
     .unwrap();
+    pub static ref FULL_COMPACT_INCREMENTAL: Histogram = register_histogram!(
+        "tikv_storage_full_compact_increment_duration_seconds",
```

Comment:
please update metrics/grafana/tikv-details.json as well

Reply:
working on that.

Reply:
done.